.gitignore

```diff
@@ -5,3 +5,4 @@ documentation/
 *swp
 *.user*
 .DS_Store
+.vscode/
```

Comment:
Such things don't belong in a project gitignore, but your private git configuration.
As vscode is a popular editor, I'd be willing to allow this here. But ideally it wasn't here.